R/RLearner_classif_binomial.R

```diff
@@ -27,6 +27,11 @@ trainLearner.classif.binomial = function(.learner, .task, .subset, .weights = NU
 
 #' @export
 predictLearner.classif.binomial = function(.learner, .model, .newdata, ...) {
+
+  # checks factor levels, and fixes eventually occuring problems due to additional
```

Comment:
I think you mean the German "eventuell" here, which would translate to "possibly" in English? The English word "eventually" means "in the end, after some time", and I feel this is not what you want to say here. Sorry to be nitpicking, but this might actually confuse people who don't know the German word.

Reply:
no worries, every comment is appreciated :smile: 
You're right, I changed it :+1: